src/style.css

```diff
@@ -1,3 +1,55 @@
-body {
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style-type: none;
+  text-decoration: none;
+  color: goldenrod;
+  transition: all 0.5s ease-out;
+}
+
+:root {
+  --flex: flex;
+  --inline: inline;
+  --black: black;
+  --primary: goldenrod;
+  --secondary: white;
+}
+
+header {
+  display: var(--flex);
+  align-items: center;
+  justify-content: space-between;
+  padding: 1% 10%;
+  background: var(--black);
+  box-shadow: rgba(56, 61, 67, 0.9) 1px 18px 14px;
+}
+
+.logo {
+  align-items: flex-start;
+  justify-content: flex-start;
+}
+
+.logo img {
+  background-color: var(--black);
+  width: 116px;
+  height: auto;
+  padding: 0;
+  align-self: flex-start;
+}
+
+.nav li {
+  display: var(--inline);
+  padding: 2rem 3rem;
+  font-size: 1.5rem;
+  font-weight: 600;
+}
+
+.nav li:hover {
   background-color: white;
```

Comment:
I suggest you reuse the variable you declared above. This ensures you adhere to CSS best practices as repetition is avoided(DRY code)

Reply:
Updated accordingly. 🙏